src/portable/windows.rs

```diff
@@ -310,20 +352,32 @@ fn get_wsl_distro(install: bool) -> anyhow::Result<Wsl> {
         fs::create_dir_all(&download_dir)?;
         let download_path = download_dir.join("debian.zip");
         task::block_on(download(&download_path, &*DISTRO_URL, false, false))?;
+        echo!("Unpacking WSL distribution...");
         let appx_path = download_dir.join("debian.appx");
         unpack_appx(&download_path, &appx_path)?;
         let root_path = download_dir.join("install.tar");
         unpack_root(&appx_path, &root_path)?;
 
         let distro_path = wsl_dir()?.join(CURRENT_DISTRO);
         fs::create_dir_all(&distro_path)?;
-        echo!("Initializing WSL...");
+        echo!("Initializing WSL distribution...");
         process::Native::new("wsl import", "wsl", "wsl")
             .arg("--import")
             .arg(CURRENT_DISTRO)
             .arg(&distro_path)
             .arg(&root_path)
             .run()?;
+        let ver = get_version(CURRENT_DISTRO)?;
+        if ver != 2 {
```

Comment:
Wouldn't it be easier to run `--import` with `--version 2`?

Reply:
Let me verify again, but I'm pretty sure my WSL cli doesn't have that option.

Reply:
Got it after windows update.